controllers/apply/form-router-next/views/dashboard.njk

```diff
@@ -2,6 +2,7 @@
 {% from "components/content-box/macro.njk" import contentBox %}
 {% from "components/icons.njk" import iconTick, iconBin %}
 {% from "components/user-status/macro.njk" import userStatus with context %}
+{% from "components/user-navigation/macro.njk" import formNavigation with context %}
```

Comment:
`formNavigation` should probably be `userNavigation` to match the directory / class names.